tests/Curators/Events/NewsArticleAboutEventAddedJSONDeserializerTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Curators\Events;
+
+use PHPUnit\Framework\TestCase;
+use ValueObjects\StringLiteral\StringLiteral;
+
+final class NewsArticleAboutEventAddedJSONDeserializerTest extends TestCase
+{
+    /**
+     * @var NewsArticleAboutEventAddedJSONDeserializer
+     */
+    private $deserializer;
+
+    public function setUp()
```

Comment:
Nitpicking here, but I think the setUp method in `TestCase` is `protected`. There's no point in loosening the access level.